proc-macros/src/api_def.rs

```diff
@@ -153,11 +153,10 @@ impl syn::parse::Parse for ApiMethod {
 impl ApiMethodAttrs {
 	/// Tries to merge another `ApiMethodAttrs` within this one. Returns an error if there is an
 	/// overlap in the attributes.
-	// TODO: span
 	fn try_merge(&mut self, other: ApiMethodAttrs) -> syn::parse::Result<()> {
 		if let Some(method) = other.method {
-			if self.method.is_some() {
-				// TODO: return Err(())
+			if self.method.as_deref() == Some(&method) {
```

Comment:
@niklasad1 if I got the description of this function right, a part of the check was missing.
I added some tests on the way. Can you confirm?

Reply:
Looks good thanks :+1: